pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -367,6 +370,147 @@ var _ = Describe("Validating Webhook", func() {
 			Expect(len(resp.Result.Details.Causes)).To(Equal(1))
 			Expect(resp.Result.Details.Causes[0].Message).To(Equal("update of VMI object is restricted"))
 		})
+		It("should allow valid VirtualMachineInstance meta on update of node label by kubevirt components", func() {
+			vmi := testutils.NewVMIReferenceWithLabels("someNode", "someVM")
+
+			updateVmi := vmi.DeepCopy()
+			updateVmi.ObjectMeta.Labels[v1.NodeNameLabel] = "someNewNodeLabel"
+
+			newVMIBytes, _ := json.Marshal(&updateVmi)
+			oldVMIBytes, _ := json.Marshal(&vmi)
+
+			kubevirtServiceAccounts := [...]string{
```

Comment:
instead of this can you use the function you create https://github.com/kubevirt/kubevirt/pull/2136/files#diff-9286ff8b0ac96e6c09b75d3bf89a47baR1608

Reply:
Thanks for noticing this!
I've updated the PR